com.unity.render-pipelines.core/Runtime/Common/ConstantBuffer.cs

```diff
@@ -19,7 +19,7 @@ public class ConstantBuffer
         /// <param name="shaderId">Shader porperty id to bind the constant buffer to.</param>
         public static void PushGlobal<CBType>(CommandBuffer cmd, in CBType data, int shaderId) where CBType : struct
```

Comment:
Might be worth adding the PushGlobal version to regular CBs as well for consistency.